numpy/ma/core.py

```diff
@@ -7378,11 +7381,11 @@ def append(a, b, axis=None):
 
     Parameters
     ----------
-    arr : array_like
+    a : array_like
         Values are appended to a copy of this array.
-    values : array_like
-        These values are appended to a copy of `arr`.  It must be of the
-        correct shape (the same shape as `arr`, excluding `axis`).  If `axis`
+    b : array_like
+        These values are appended to a copy of `a`.  It must be of the
```

Comment:
Just found out that PEP8 mandates two spaces after a period ending a sentence, so you actually have it right. We are terribly inconsistent about this though...